api/resmap/reswrangler.go

```diff
@@ -88,6 +88,19 @@ func (m *resWrangler) append(res *resource.Resource) {
 	m.rList = append(m.rList, res)
 }
 
+// AnnotateAll implements ResMap
+func (m *resWrangler) AnnotateAll(key string, value string) error {
+	return m.ApplyFilter(annotations.Filter{
+		Annotations: map[string]string{
+			key: value,
+		},
+		FsSlice: []types.FieldSpec{{
+			Path:               "metadata/annotations",
+			CreateIfNotPresent: true,
+		}},
+	})
+}
```

Comment:
is there a reason to move this?

Reply:
no, it just happened as a result of the rebase (the origin annotation PR also added this method to resmap)

Reply:
I moved it back